tests/components/bond/test_init.py

```diff
@@ -0,0 +1,56 @@
+"""Tests for the Bond module."""
+from homeassistant.components.bond.const import DOMAIN
+from homeassistant.config_entries import ENTRY_STATE_LOADED, ENTRY_STATE_NOT_LOADED
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+
+async def test_async_setup_no_domain_config(hass: HomeAssistant):
+    """Test setup without configuration is noop."""
+    result = await async_setup_component(hass, DOMAIN, {})
+
+    assert result is True
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)